components/application-operator/charts/application/templates/virtual-service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ .Release.Name }}-event-service
```

Comment:
This `VirtualService` is not only for the `event-service` but also for Application Registry so I think it would be better to stay with `{{ .Release.Name }}` as its name.

Reply:
We want a HTTPSource (or MQTTSource, or whatever we implement in the future) to represent the source of all events for a given Application, not sure if adding extra noise to the object name would clearly convey that idea.

I'm ok to find a better name for the current VirtualService though.

Btw, how can a single VirtualService front both the event-service _and_ the registry? (maybe take that discussion to the community Slack)

Reply:
I renamed the VirtualService to `myapp-validator`. What do you think?

Reply:
Fine for me :)